src/Aspire.Hosting.Milvus/MilvusContainerImageTags.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.Milvus;
+
+internal static class MilvusContainerImageTags
+{
+    public const string Registry = "docker.io";
+    public const string Image = "milvusdb/milvus";
+    public const string Tag = "v2.3.15";
```

Comment:
Based on our policy here you can probably use `2.3-latest` here.

Reply:
Looks like there is a `2.4-latest` now. https://hub.docker.com/r/milvusdb/milvus/tags

Reply:
Only reason for now I've kept it is the client versioning in the docs mentions staying in sync with the server versioning.  e.g., the client is 2.3.x targets the 2.3.x version of the server.  It says it is compatible, but that's the only reason so far.  IMO the Milvus tagging is all over the place.

Reply:
Changing to 2.3-latest